packages/patternfly-4/react-core/src/components/Wizard/Wizard.tsx

```diff
@@ -122,12 +125,17 @@ export class Wizard extends React.Component<WizardProps, WizardState> {
   private container: HTMLDivElement;
   private titleId: string;
   private descriptionId: string;
+  private isModal: boolean;
 
   constructor(props: WizardProps) {
     super(props);
     const newId = Wizard.currentId++;
-    this.titleId = `pf-wizard-title-${newId}`;
-    this.descriptionId = `pf-wizard-description-${newId}`;
+    this.isModal = !props.isInPage;
```

Comment:
user cannot should not have an ability to change the inpage wizard to modal one after it is constructed